docs/src/use/configure/migration-guide.md

```diff
@@ -0,0 +1,422 @@
+---
+title: Configuration Migration Guide
+eleventyNavigation:
+    key: migration guide
+    parent: configure
+    title: Configuration Migration Guide
+    order: 8
+---
+
+This guide provides an overview of how you can migrate your ESLint configuration file from the `.eslintrc` format to the new configuration file format, `eslint.config.js`.
+
+To learn more about the `eslint.config.js` file format, refer to [this blog post](https://eslint.org/blog/2022/08/new-config-system-part-2/).
+
+For reference information on these configuration formats, refer to the following documentation:
+
+* [`.eslintrc` configuration files](configuration-files)
+* [`eslint.config.js` configuration files](configuration-files-new)
+
+## Start Using `eslint.config.js`
+
+Starting with ESLint v9.0.0, the `eslint.config.js` file format will be the default configuration file format. Once ESLint v9.0.0 is released, you can start using the `eslint.config.js` file format without any additional configuration.
+
+To use `eslint.config.js` with ESLint v8, place a `eslint.config.js` file in the root of your project **or** set an `ESLINT_USE_FLAT_CONFIG` environment variable to `true`.
+
+## Key Differences between Configuration Formats
+
+A few of the most notable differences between the `.eslintrc` and `eslint.config.js` formats are the following:
+
+### Importing Plugins and Custom Parsers
+
+#### .eslintrc
```

Comment:
I think it would be helpful to summarize the difference under each header and then show the examples without headers. Maybe something like:

> In eslintrc, plugins are loaded by specifying the name of the plugin in the `plugins` array; in flat config, you'll need to import the plugin object and then assign it to a property in the `plugins` object.